packages/v4/patternfly-docs/content/accessibility/alert/alert.md

```diff
@@ -19,21 +21,27 @@ At a minimum, an alert should meet the following criteria:
   <ListItem>
     <Checkbox id="alert-a11y-checkbox-1" label="If an alert will dynamically appear or update, it is rendered inside of an alert group component." />
   </ListItem>
+  <ListItem>
+    <Checkbox id="alert-a11y-checkbox-2" label="If an alert is using a heading level for its title, the heading level should fit the page heading level order so that heading levels are not skipped." />
```

Comment:
```suggestion
    <Checkbox id="alert-a11y-checkbox-2" label="If an alert is using a heading element for its title, the heading level fits the page heading level order so that heading levels are not skipped." />
```